web/src/main/java/com/graphhopper/countries/GermanySpatialRule.java

```diff
@@ -15,17 +15,18 @@
  *  See the License for the specific language governing permissions and
  *  limitations under the License.
  */
-package com.graphhopper.routing.util.spatialrules.countries;
+package com.graphhopper.countries;
```

Comment:
Yes I know, but why? This should be a core feature. For example if I want to use it from the MiniGraphUI (just one example, think of any usage that does not need the web module), I would need to add a dependency to the web module. That seems wrong to me.